src/main/java/com/openlattice/chronicle/controllers/ChronicleStudyController.java

```diff
@@ -124,7 +129,23 @@ public Boolean isKnownParticipant(
             @RequestParam( value = FILE_TYPE, required = false ) FileType fileType,
             HttpServletResponse response ) {
 
-        setContentDisposition( response, participantEntityId.toString(), fileType );
+        String participantId = chronicleService
+                .getParticipantEntity( studyId, participantEntityId )
```

Comment:
is the second parameter the entityId or the entityKeyId? We shouldn't be using entity ids here I think, since this is an internal dataset...

Reply:
It's a `UUID`, so I guess it's the `entityKeyId`.